<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device assistant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayoutPort">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutPortName">
        <item>
         <widget class="QLabel" name="labelPortName">
          <property name="text">
           <string>Port Name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxPortName">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutBaudRate">
        <item>
         <widget class="QLabel" name="labelBaudRate">
          <property name="text">
           <string>Baud Rate :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBaudRate">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPortConnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDeviceState">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowserLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLogClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabConfig">
       <attribute name="title">
        <string>Config</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabDownload">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Download</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutRecent">
          <item>
           <widget class="QRadioButton" name="radioButtonRecent">
            <property name="text">
             <string>Recent data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRecentPackFrom">
            <property name="text">
             <string>From packet:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRecentPackFrom"/>
          </item>
          <item>
           <widget class="QLabel" name="labelRecentPackTo">
            <property name="text">
             <string>To packet:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRecentPackTo"/>
          </item>
          <item>
           <spacer name="horizontalRecent">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRecentRequest">
            <property name="text">
             <string>Request</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutHistoric">
          <item>
           <widget class="QRadioButton" name="radioButtonHistoric">
            <property name="text">
             <string>Historical data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHistoricDateTime">
            <property name="text">
             <string>Date time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEditHistoric"/>
          </item>
          <item>
           <widget class="QLabel" name="labelHistoricPacketFrom">
            <property name="text">
             <string>From packet:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxHistoricPacketFrom"/>
          </item>
          <item>
           <widget class="QLabel" name="labelHistoricPacketTo">
            <property name="text">
             <string>To packet:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxHistoricPacketTo"/>
          </item>
          <item>
           <spacer name="horizontalSpacerHistoric">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonHistoricRequest">
            <property name="text">
             <string>Request</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutType">
          <item>
           <widget class="QLabel" name="labelType">
            <property name="text">
             <string>Download type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelTypeSensor">
            <property name="text">
             <string>Sensor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxTypeSensor">
            <item>
             <property name="text">
              <string>Accel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gyro</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Angle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Adc 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Adc 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Accel result</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelTypeData">
            <property name="text">
             <string>Data:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxTypeData">
            <item>
             <property name="text">
              <string>Psd</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Statistic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raw</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerType">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTypeRequest">
            <property name="text">
             <string>Request</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDownload">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1398</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutApplication"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuPort"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAboutApplication">
   <property name="text">
    <string>Application</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
